libs/labelbox/src/labelbox/schema/tool_building/classification.py

```diff
@@ -0,0 +1,367 @@
+import warnings
```

Comment:
Moved these classes to a separate package to avoid circular dependencies

Reply:
Good!